package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "probot-app-pr-label",
+  "version": "1.0.0",
+  "description": "GitHub App that labels release PRs",
```

Comment:
I think this is meant to be something else?

Reply:
Fixed!